R/covariance_plot.R

```diff
@@ -65,3 +65,39 @@ h_get_timepoint_vars <- function(vcov_matrix,
     col_time = col_time
   )
 }
+#' @description This function vectorizes the upper-diagonal elements of a symmetric matrix (e.g. the covariance matrix)
```

Comment:
```suggestion

#' @description This function vectorizes the upper-diagonal elements of a symmetric matrix (e.g. the covariance matrix)
```

(just to have some space between end of last function and this)